common/djangoapps/track/tests/test_middleware.py

```diff
@@ -145,3 +145,21 @@ def test_request_headers(self):
             'ip': ip_address,
             'agent': user_agent,
         })
+
+    @patch('track.middleware.TRUNCATION_LENGTH', 6)
+    def test_student_answer_request(self):
+        request = self.request_factory.post('/dummy')
+        request.POST = {'student_answer': ['firstanswer', 'secondanswer']}
+        self.track_middleware.process_request(request)
+
+        event = '{"POST": {"student_answer": ["fir", "sec"]}, "GET": {}}'
+        self.mock_server_track.assert_called_with(request, request.META['PATH_INFO'], event)
+
+    @patch('track.middleware.TRUNCATION_LENGTH', 20)
+    def test_submission_request(self):
+        request = self.request_factory.post('/dummy')
+        request.POST = {'{"submission":["first open response", "second open response"]}': [] }
+        self.track_middleware.process_request(request)
+
```

Comment:
Kind of a nit but not really: I really don't care for these superfluous newlines here. Given that PEP8 dictates we place a single empty line between class methods, this is rather difficult to follow visually; there's the same amount of space in the _middle_ of each of these methods as there is _between_ each.

Can we drop this blank line?